CONTRIBUTING.md

```diff
@@ -0,0 +1,88 @@
+# Keen IO Python Client
+
+First off, thank you for considering contributing to this official Keen IO client. It's people like you that make Keen IO such a great tool.
+
+We put these guidelines together to try and make working with our SDK as straight forward as possible, and hopefully help you understand how we communicate about potential changes and improvements.
+
+Improving documentation, bug triaging, building modules for various frameworks or writing tutorials are all examples of helpful contributions we really appreciate.
+
+Please, don't use the issue tracker for support questions. If you have a support question please come hang out in our Slack  at http://keen.chat or send an email to team@keen.io.
+
+## Guidelines
```

Comment:
I think when keen-js got updated, the newer version of this section might have gotten lost. Here's the updated version to include: https://github.com/keen/keen-js/blob/v3/CONTRIBUTING.md#guidelines

We have an awesome Keen IO Community CoC to include.